lib/trento_web/plugs/unplug_charts_enabled_predicate.ex

```diff
@@ -0,0 +1,8 @@
+defmodule TrentoWeb.Plugs.UnplugChartsEnabledPredicate do
```

Comment:
Maybe it would be better to organize these predicates in its own folder.
`/plugs/unplug/predicates/charts_enabled.ex`
`defmodule TrentoWeb.Plugs.Unplug.Predicates.ChartsEnabled`

Reply:
Besides that, I would rename it to `ChartsDisabled` as it is true when they are disabled.
Otherwise, the usage in the router looks contradictory, as it uses the Disabled plug with the enabled predicate.

Reply:
nice catch! done